### PR TITLE
Add postera api

### PR DESCRIPTION
**Pull request by laurenreid1**
## Description
postera_api package added to provide command line interface for interacting with PostEra API. 

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - Functionality to create, read or update a PostEra molecule set - DONE
  - Format designed to add new PostEra API endpoints in the future if needed

## Questions
- unit tests have been added in postera_api.tests - do these needed to be added to a global unit test suite?

## Status
- [ ] Ready to go

### CODE REVIEW

**Comment by mikemhenry** [comment]
Tagging in @emripka 

I need to fix the permissions to add you as a reviewer, stay tuned!

**Comment by mikemhenry** [comment]
@emripka invite sent!

**Review by JenkeScheen — APPROVED** [review]
LGTM, one small comment though: we try to use [`black`](https://pypi.org/project/black/) for code formatting (BK/AP in pycharm, I just use the conda install). Very straightforward to install and apply and makes the code a lot more readable!

**Comment by dotsdl** [comment]
@emripka could you give this a review? I think that will be most useful coming from you at PostEra!

File: asapdiscovery/postera_api/__main__.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
+from src.arg_parser import arg_parser
+from src.molecule_set_crud import MoleculeSetCRUD, MoleculeList, MoleculeUpdateList
+
+import pandas as pd
+
+
+def molecule_set(args):
+    """Handle molecule set commands"""
```
[the remark this answers is not in the document]
Comment (laurenreid1) [reply asapdiscovery/postera_api/__main__.py]: Thanks! 

`black` installed and applied

added more info to the doc string

**Pull request merged**